test/test_stacked_bar.rb

```diff
@@ -9,50 +9,66 @@ def setup
       [:Charles, [80, 54, 67, 54]],
       [:Julie, [22, 29, 35, 38]]
     ]
-    @sample_labels = {
-      0 => '5/6',
-      1 => '5/15',
-      2 => '5/24'
-    }
   end
 
   def test_bar_graph
-    g = Gruff::StackedBar.new
+    @datasets = [
+      [:Jimmy, [25, 36, 86, 39]],
+      [:Charles, [80, 54, 67, 54]],
+      [:Julie, [2, 29, 35, 38]]
+    ]
+    g = setup_basic_graph
```

Comment:
Refactored the tests to use the setup_basic_graph pattern seen in other test sets.